MAINTAINERS.md

```diff
@@ -8,3 +8,4 @@ Maintainership is on a per project basis.
   - Colin Sullivan <colin@nats.io> [@ColinSullivan1](https://github.com/ColinSullivan1)
   - Derek Collison <derek@nats.io> [@derekcollison](https://github.com/derekcollison)
   - Ivan Kozlovic <ivan@nats.io> [@kozlovic](https://github.com/kozlovic)
+  - Scott Fauerbach <scott@nats.io> [@scottf](https://github.com/scottf)
```

Comment:
Unfortunately we can't do this right now.  We need a formal vote to add you as a Maintainer per NATS Governance.  We'll certainly revisit this!

Reply:
ok sorry, will remove